docker/liberica-jre/Dockerfile

```diff
@@ -4,12 +4,28 @@ FROM ghcr.io/permutive-engineering/permutive-liberica-base
 
 ARG LIBERICA_VERSION=""
 ARG LIBERICA_VARIANT=jre
-ARG LIBERICA_SHA1=""
+ARG LIBERICA_SHA1_ARM=""
+ARG LIBERICA_SHA1_X86=""
```

Comment:
**question:** Do you need to define as empty these so they can be passed in to the image?

Potentially just defining them for clarity?

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org/) standard</sub>

Reply:
Yes, they have to be there so that they can be passed at build time